RELEASE/scripts/autoscend/auto_post_adv.ash

```diff
@@ -421,7 +421,12 @@ boolean auto_post_adventure()
 			use_skill(1, $skill[Soul Rotation]);
 		}
 		int missing = (my_maxmp() - my_mp()) / 15;		//soul food restores 15 MP per cast.
-		int casts = min(missing, my_soulsauce() / 5);	//soul food costs 5 soulsauce per cast.
+		int availableSauce = my_soulsauce();
+		if(availableSauce >= 5 && my_mp() > 100 && my_mp() > 0.8*my_maxmp())
```

Comment:
To be more accurate, my_mp() should also have however many casts of soul food we can do (while still reserving 5 for bubble)

I think 100 and > 80% are good enough criteria's for now. Perhaps can be improved later when we know how much mana we need for critical buffs, which we'll hopefully get when #881 is fixed